app/views/promotion/landingPage.scala.html

```diff
@@ -0,0 +1,77 @@
+@import com.github.nscala_time.time.Imports.DateTime
+@import com.gu.memsub.promo.PromoCode
+@import com.gu.memsub.promo.Promotion.AnyPromotion
+@import com.gu.subscriptions.DigipackCatalog
+@import model.DigitalEdition
+@import views.support.Dates._
+@(edition: DigitalEdition, catalog: DigipackCatalog, promoCode: PromoCode, promotion: AnyPromotion)
+
+@main(
+    title = "The Digital Pack offer | Subscriptions | The Guardian",
+    bodyClasses = Seq("is-wide")
+) {
+    <main class="promo-landing-page">
+        <section class="page-container gs-container gs-container--slim">
+            <div class="page-slice">
+                <div class="page-slice__content">
+                @fragments.page.header("The Digital Pack offer", None)
+                </div>
+            </div>
+        </section>
+        <section class="section--grey">
+            <div class="page-container gs-container gs-container--slim page-slice">
+                @fragments.promotion.digipack(edition, catalog, promoCode, promotion)
+            </div>
+        </section>
+        <section class="page-container gs-container gs-container--slim">
+            <div class="page-slice">
+                <div class="page-slice__content">
+                    <h2 class="promo-landing-page__section-head">Discover the Guardian Daily Edition</h2>
+                    @fragments.promotion.discover(edition, catalog, promoCode, promotion)
+                </div>
+            </div>
+        </section>
+        <section class="section--blue">
```

Comment:
this a modifier class. it's supposed to be used in conjunction with the base class. https://github.com/guardian/membership-frontend/blob/master/docs/client-side-principles.md#component-modifiers